### ISSUE TITLE
Num lock/Caps lock labels are not centered

### ISSUE THREAD

**Issue by lenemter** [elementary/greeter#699]
### What Happened?

![bug](https://github.com/elementary/greeter/assets/80143868/a9c62ae0-fbd6-4e61-b508-8078dc4254df)

### Steps to Reproduce

1. Use OS 7/8 early access
2. Open lock screen

### Expected Behavior

Centered labels

### OS Version

Early Access

### Software Version

Compiled from git

**Comment by ryonakano** [comment]
Makes sense, but this might be by design, because the target that Num Lock applied is only the password input, and the gear icon is not affected at all.
